### ISSUE TITLE
 HTML 5.0 Report: Add method and class of the failing test

### ISSUE THREAD

**Issue by lgoltz** [opengeospatial/teamengine#286]
If  the ETS is implemented with TestNG the method name and class should be reported in the details of a test. In the old report also a link to the JavaDoc is available.

**Comment by keshav-nangare** [comment]
@lgoltz  @dstenger 

If we have to add the method name and class in details of the test then first we need to add it in earl-result.rdf then we can show in HTML report.

How we can represent the method and class in the EARL?

**Comment by lgoltz** [comment]
@keshav-nangare Can you make a proposal how to enhance the EARL?

**Comment by keshav-nangare** [comment]
Currently we have the one attribute in earl which is include the class path as well as method name, maybe we can easily extract the class name and method name from that

```
Example: 
<earl:TestCase rdf:about="org/opengis/cite/iso19136/general/GeneralSchemaTests#declareTargetNamespace">

Class Name: GeneralSchemaTests
Method name: declareTargetNamespace
```

If it is fine then we can add class name and method name in to the html report

**Comment by lgoltz** [comment]
@keshav-nangare Thats fine for me!

**Comment by keshav-nangare** [comment]
Created the dummy template test details page including class and method name and link to javadoc.

![added_class_method_name](https://user-images.githubusercontent.com/14732650/34286494-fc28389e-e706-11e7-97d9-ba5497914772.jpg)

Please let me know if there is any changes.

**Comment by keshav-nangare** [comment]
I will add new changes to test details page and will do further changes if required.

**Comment by keshav-nangare** [comment]
Add method and class of failing test if the test-suite is based on the TestNG. But we can't recognize  whether the suite is based of CTL or TestNg test from the EARL report.  
To Identify the test-suite we need to add the custom CITE earl property in earl.rdf result.

**Comment by keshav-nangare** [comment]
#287

**Comment by keshav-nangare** [comment]
Added test and class into the test detail page of failing test if the ets is implemented with the TestNG.

Fixed with this commit: 7d7a2ba948d366c7e0baa63e032f991367627104 [PR: 289]

**Comment by dstenger** [comment]
## Test

Start TEAM Engine via Docker environment [1].

Execute test run with ETS WFS 2.0 [2].

For example, when a test fails following is added to test description:
```
Class Name:BBOXTests
Method Name:invalidGeometryOperand
Path:org/opengis/cite/iso19142/basic/filter/spatial/BBOXTests 
```

[1] https://github.com/opengeospatial/teamengine-docker
[2] http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wfs200?service=WFS&request=GetCapabilities
